torchtnt/utils/lr_scheduler.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# This PR exposes LRScheduler as a public class
+# https://github.com/pytorch/pytorch/pull/88503
+try:
```

Comment:
this is missing the torch import, causing the unit test failures

Reply:
Added https://github.com/pytorch/tnt/pull/286/files#diff-0c2359f3fea16dad843c05e931f77983ce530bca24b7bdde1f5340eeabe12057R7